api/src/main/java/uk/gov/hmcts/fees2/register/api/controllers/mapper/FeeDtoMapper.java

```diff
@@ -63,6 +63,7 @@ private void fillFee(FeeDto request, Fee fee, String author) {
 
         FeeVersion version = toFeeVersion(request.getVersion(), author);
         version.setFee(fee);
+        version.setReasonForUpdate(request.getReasonForUpdate());
```

Comment:
Why are we persisting a Free Text from UI without a XSS Input Encoding ?

Reply:
I pushed now, it was in my local